dataprofiler/labelers/character_level_cnn_model.py

```diff
@@ -503,7 +547,7 @@ def encoding_function(input_str: tf.Tensor) -> tf.Tensor:
             if self._parameters["dropout"]:
                 self._model.add(tf.keras.layers.Dropout(self._parameters["dropout"]))
             # Add batch normalization, set fused = True for compactness
```

Comment:
```suggestion
            # Add batch normalization
```

Reply:
no need for extra part of comment now

Reply:
Just removed it entirely since the line essentially would be the comment